h5py/tests/common.py

```diff
@@ -66,35 +63,33 @@ def mktemp(self, suffix='.hdf5', prefix='', dir=None):
         if dir is None:
             dir = self.tempdir
         return tempfile.mktemp(suffix, prefix, dir=self.tempdir)
-        
+
     def setUp(self):
         self.f = h5py.File(self.mktemp(), 'w')
-        
+
     def tearDown(self):
         try:
             if self.f:
                 self.f.close()
         except:
             pass
 
-    if not hasattr(ut.TestCase, 'assertSameElements'):
-        # shim until this is ported into unittest2
-        def assertSameElements(self, a, b):
-            for x in a:
-                match = False
-                for y in b:
-                    if x == y:
-                        match = True
-                if not match:
-                    raise AssertionError("Item '%s' appears in a but not b" % x)
-
-            for x in b:
-                match = False
-                for y in a:
-                    if x == y:
-                        match = True
-                if not match:
-                    raise AssertionError("Item '%s' appears in b but not a" % x)
+    def assertSameElements(self, a, b):
```

Comment:
From the comment it looks like this code was meant to be temporary and now we have just removed the conditional.  Is there no chance of backport or are we just ove-riding no matter what to be on the safe side?